src/components/formik-toggle/FormikToggle.tsx

```diff
@@ -166,40 +160,30 @@ export class FormikToggle extends React.Component<Props> {
       toggleContent = (
         <Div>
           <Icon>{this.props.tooltipToggle}</Icon>
-          <LabelBeforeToffle>{this.props.label}</LabelBeforeToffle>
+          <LabelBeforeToggle>{this.props.label}</LabelBeforeToggle>
           <Icon>{this.props.iconRight}</Icon>
           <Toggle>
-            <Slider
-              error={this.props.error}
-              disabled={this.props.disabled}
-              focussed={this.props.focussed}
-            />
             <Input
               {...this.props.field}
               type="checkbox"
               disabled={this.props.disabled}
-              error={this.props.error}
             />
+            <span></span>
           </Toggle>
         </Div>
       );
     }
     if (!hasTooltip) {
       toggleContent = (
         <Div>
-          <LabelBeforeToffle>{this.props.label}</LabelBeforeToffle>
+          <LabelBeforeToggle>{this.props.label}</LabelBeforeToggle>
           <Toggle>
-            <Slider
-              error={this.props.error}
-              disabled={this.props.disabled}
-              focussed={this.props.focussed}
-            />
             <Input
               {...this.props.field}
               type="checkbox"
               disabled={this.props.disabled}
-              error={this.props.error}
             />
+            <span></span>
```

Comment:
Lege span?

Reply:
Als het voor die ``` input:checked + span  ``` selector is kan je beter een duidelijke className gebruiken zodat je ineens ziet waar het voor dient. Of een comment erbij zetten. Anders gaat iemand (wss ikzelf 🦆) in de toekomst dit ooit verwijderen omdat hij denkt dat dit niets doet :D